src/backend/window.rs

```diff
@@ -35,6 +35,12 @@ pub struct Window {
 	/// The image to display (if any).
 	pub image: Option<GpuImage>,
 
+	/// The zoom of the image.
+	pub zoom: f32,
+
+	/// The pan of the image.
+	pub pan: [f32; 2],
```

Comment:
I'm not sure if I prefer `pan` or `translation`, but in any case the documentation should be clear how this is interpreted. Is it an offset applied to the image or to the viewport?

Reply:
I added a comment to clarify what it means.

Reply:
I added a comment to clarify what it means.

Reply:
I would make it even more explicit: mention that a positive X value moves the image to the right, and a positive Y value moves it down.